assets/src/blocks/Articles/ArticlesEditor.js

```diff
@@ -81,12 +79,12 @@ const renderEdit = (attributes, toAttribute) => {
           {attributes.tags.length === 0 && attributes.post_types.length === 0 &&
             <div>
               <hr />
-              <label>{__('Manual override', 'planet4-blocks-backend')}</label>
+              <label htmlFor>{__('Manual override', 'planet4-blocks-backend')}</label>
```

Comment:
Maybe instead of adding empty `htmlFor` attributes, we shouldn't be using labels for this? A paragraph for example would probably be fine